node/config.js

```diff
@@ -15,6 +15,10 @@ exports.getConfig = async (configFile) => {
   logger.msgOk('Loading cy-runner configuration')
   let config = storage.loadConfig(configFile)
 
+  // Check mixed paths and missing specs
+  logger.msgOk('Checking mixed paths on specs')
```

Comment:
In logs, it would be helpful if we replace
Checking mixed paths on specs -> Checking mixed paths/missing specs in cy-runner.yml file